interface/billing/billing_report.php

```diff
@@ -1065,8 +1065,8 @@ class='link_submit'  ><?php echo '['.xlt('End Of Day Report').']' ?></a>
     if ($iter['id']) $rhtml .= text(oeFormatSDFT(strtotime($iter{"date"})));
     $rhtml .= "</span></td>\n";
     if ($iter['id'] && $iter['authorized'] != 1) {
-      $rhtml .= "<td><span class=alert>".xlt("Note: This code was not entered by an authorized user. Only authorized codes may be uploaded to the Open Medical Billing Network for processing. If you wish to upload these codes, please select an authorized user here.")."</span></td>\n";
-    }
+       $rhtml .= "<td><span class=alert>".xlt("Note: This code is unauthorized.")."</span></td>\n";
```

Comment:
Why will it be different for different users? (just need examples to figure this out)